thirdeye/thirdeye-frontend/app/styles/components/te-anomaly-table.scss

```diff
@@ -0,0 +1,149 @@
+.te-anomaly-table {
+  width: 100%;
+  margin-top: 12px;
+  border: 1px solid app-shade(black, 1);
+
+  &__wrapper {
+    height: 500px;
+    overflow-y: scroll;
+  }
+
+  &__text {
+    margin-right: 30px;
+  }
+
+  &__head {
+    line-height: 40px;
+  }
+
+  &__cell {
+    padding: 5px 20px;
+    font-size: 15px;
+    font-weight: 500;
+    color: app-shade(black, 8);
+    position: relative;
+
+    &:last-child {
+      max-width: 90px;
+      text-align: center;
+      padding: 4px 0 0 0;
+      vertical-align: baseline;
+    }
+  }
+
+  &__cell-head {
+    font-size: 15px;
+    font-weight: bold;
+    color: app-shade(black, 6);
+    padding-left: 20px;
+    line-height: 45px;
+
+    &--fixed {
+      width: 215px;
+    }
+  }
+
+  &__cell-link {
+    color: app-shade(black, 6);
+    white-space: nowrap;
+    font-weight: 700;
+
+    &:hover {
+      text-decoration: none;
+      cursor: pointer;
+      color: black;
+    }
+  }
+
+  &__row {
+    border-bottom: 1px solid app-shade(black, 1);
+  }
+
+  &__list {
+    list-style: none;
+    margin-left: -39px;
+    margin-bottom: 0;
+    max-width: 250px;
+  }
+
+  &__list-item {
+    &--smaller {
+      font-size: 12px;
+      margin-left: -2px;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+    &--lighter {
+      font-size: 12px;
+      color: app-shade(black, 6);
+    }
+
+    &--warning {
+      color: $te-red;
+      font-size: 13px;
+    }
+
+    &--shadow {
+      position: absolute;
+      color: app-shade(black, 2);
+      margin-left: -18px;
+    }
+  }
+
+  &__value-label {
+    font-size: 13px;
+
+    &--up {
+      color: $te-green;
+    }
+
+    &--down {
+      color: $te-red;
+    }
+  }
+
+  &__select {
+    display: inline-block;
+    margin-right: 30px;
+    width: 180px;
+  }
+
+  &__link-wrapper {
+    display: inline-block;
+    vertical-align: top;
+    margin-right: 20px;
+    padding-top: 7px;
+  }
+
+  &__link {
+    vertical-align: top;
+    color: $te-link-blue;
+    font-weight: 600;
+    letter-spacing: .03em;
+    font-size: 15px;
+  }
+
+  &__icon {
+    font-size: 14px;
+    margin-left: 5px;
+
+    &--status {
+      -webkit-animation: fadeinout 4s linear forwards;
+      animation: fadeinout 4s linear forwards;
+      position: absolute;
+      font-size: 20px;
+      margin: 14px 0 0 -26px;
+    }
+
+    &--small {
+      font-size: 11px;
+      margin-left: 0px;
+    }
+  }
+
+  &--performance {
+    margin-top: 30px;
+  }
+
+}
```

Comment:
newline please

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/
https://robots.thoughtbot.com/no-newline-at-end-of-file